test/unittests/NDB_BVL_Instrument_Test.php

```diff
@@ -1774,7 +1783,6 @@ function testToJsonParseSmartyDateTypeNotLorisForm()
             ],
             'NoResponse' => true
         ];
-        $this->_instrument->form = new \Candidate();
```

Comment:
After adding the type hint phan picked up on this too.. a candidate isn't a LorisForm. This is clearly wrong (and unused.)